src/main/java/gui/RootPane.java

```diff
@@ -45,7 +45,7 @@ public void start(Stage primaryStage) throws Exception {
         // created. Also has some useful settings.
         //TODO: change this line of code.
         Scene scene = new Scene(topLevelPane);
-        scene.getStylesheets().add("testcss.css");
+        scene.getStylesheets().add("stylesheet.css");
         primaryStage.setScene(scene);
         primaryStage.setTitle("Hoi ben een titel lol.");
```

Comment:
Could you change this to a more suitable title and maybe make a constant to make it easy accesible/tweakable